airbyte-integrations/connectors/source-gocardless/source_gocardless/spec.yaml

```diff
@@ -0,0 +1,49 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  "$schema": http://json-schema.org/draft-07/schema#
+  title: Gocardless Spec
+  type: object
+  required:
+  - access_token
+  - gocardless_environment
+  - gocardless_version
+  - start_date
+  properties:
+    access_token:
+      title: Access Token
+      type: string
+      pattern: "^(sandbox|live)_.+$"
+      description: Gocardless API TOKEN
+      airbyte_secret: true
+    gocardless_environment:
+      title: Gocardless API Environment
+      type: string
+      pattern: "^(live|sandbox)$"
+      description: |
+        Environment you are trying to connect to. Should be either `live` or
+        `sandbox`.
+    gocardless_version:
+      title: GoCardless API Version
+      type: string
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+      description: |
+        GoCardless version. This is a date. You can find the latest here: 
+        https://developer.gocardless.com/api-reference/#api-usage-making-requests
+    start_date:
+      title: Start Date
+      type: string
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
+      description: |
+        UTC date and time in the format 2017-01-25T00:00:00Z. Any data
+        before this date will not be replicated.
+      examples:
+      - '2017-01-25T00:00:00Z'
+    lookback_window_days:
```

Comment:
@girarda this parameter is used internally for the low-code cdk?

Reply:
This is a relic from a previous version of this connector I had running. I just forgot to remove it. That has now been done.

Reply:
Awesome